plugin/pxr/doc/README.md

```diff
@@ -105,7 +105,8 @@ Short flag | Long flag | Type | Default | Description
 `-mt` | `-mergeTransformAndShape` | bool | true | Combine Maya transform and shape into a single USD prim that has transform and geometry, for all "geometric primitives" (gprims). This results in smaller and faster scenes. Gprims will be "unpacked" back into transform and shape nodes when imported into Maya from USD.
 `-ro` | `-renderableOnly` | noarg |  | When set, only renderable prims are exported to USD.
 `-rlm` | `-renderLayerMode` | string | defaultLayer | Specify which render layer(s) to use during export. Valid values are: `defaultLayer`: Makes the default render layer the current render layer before exporting, then switches back after. No layer switching is done if the default render layer is already the current render layer, `currentLayer`: The current render layer is used for export and no layer switching is done, `modelingVariant`: Generates a variant in the `modelingVariant` variantSet for each render layer in the scene. The default render layer is made the default variant selection.
-`-shd` | `-shadingMode` | string | `displayColor` | Set the shading schema to use. Valid values are: `none`: export no shading data to the USD, `displayColor`: unless there is a colorset named `displayColor` on a Mesh, export the diffuse color of its bound shader as `displayColor` primvar on the USD Mesh, `pxrRis`: export the authored Maya shading networks, applying the same translations applied by RenderMan for Maya to the shader types.
+`-shd` | `-shadingMode` | string | `displayColor` | Set the shading schema to use. Valid values are: `none`: export no shading data to the USD, `displayColor`: unless there is a colorset named `displayColor` on a Mesh, export the diffuse color of its bound shader as `displayColor` primvar on the USD Mesh, `pxrRis`: export the authored Maya shading networks, applying the same translations applied by RenderMan for Maya to the shader types, `useRegistry`: Use a registry based to export the Maya shading network to an equivalent UsdShade network.
+`-rdc` | `-renderContext` | string | `preview` | Selects a render context for export. The default value `preview` will export to a UsdPreviewSurface shading netword. A plugin mechanism allows more render contexts to be registered. Use the `mayaUSDListShadingModes` command to explore the possible options.
```

Comment:
typo: network

And sorry to keep harping on it, but "preview" is not a render context. I'd again advocate for "universal". We're authoring UsdPreviewSurface since we expect that to be consumable by virtually all renderers.